src/NumericalAlgorithms/Interpolation/InterpolationTarget.hpp

```diff
@@ -94,6 +96,19 @@ namespace intrp {
 ///      Domain will be filled with this value. If this variable is not defined,
 ///      then the `apply` function must check for invalid points,
 ///      and should typically exit with an error message if it finds any.
+/// - should_interpolate (optional):
+///      A template function that takes as a single argument a const reference
+///      to the \ref DataBoxGroup, and returns a `bool`: `true` if the
+///      element should interpolate in the current state, `false` otherwise.
+///      Often, this will examine the `temporal_id` to determine whether to
+///      interpolate, but it may examine any part of the \ref DataBoxGroup.
+///      If omitted, the behavior is as if the `should_interpolate` function
+///      always returns true. This function is called using the DG element's
+///      \ref DataBoxGroup and is only used if `InterpolateToTarget` is
+///      in the element's action list.
+/// - interpolating_dg_component (required if not `is_sequential`):
```

Comment:
Maybe "(required if not `is_sequential`)" should be changed to "(used only if not `is_sequential`)"?